src/pages/code-for-good.js

```diff
@@ -0,0 +1,317 @@
+                    </ArticleH2>
+                    <BodyText>
+                        Use the context of a real-world application to
+                        understand MongoDB features and principles such as:
+                        schema modeling, Realm Sync, serverless logic with
+                        functions and triggers, full-text search, Charts data
+                        visualization.
+                    </BodyText>
+
+                    <StyledP>Blog posts coming soon</StyledP>
+                </UnderstandTextSection>
+
+                <StyledDashBoardImage
+                    src={WildAidDashboard}
+                    alt="A sample dashboard for the Wild Aid application"
+                />
+
+                <DashboardCaptionText>
+                    With MongoDB Charts, the O-FISH web app allows WildAid to
+                    gain insights about marine protection enforcement efforts.
+                </DashboardCaptionText>
+
+                <VesselAppContainer>
+                    <div>
+                        <VesselAppTextContainer>
+                            <ArticleH2>
+                                Build your own fully functional app for free
+                            </ArticleH2>
+                            <BodyText>
+                                All you need is a{' '}
+                                <StyledParagraphLink href="https://cloud.mongodb.com">
+                                    free Atlas account
+                                </StyledParagraphLink>{' '}
+                                and{' '}
+                                <StyledParagraphLink href="https://docs.mongodb.com/manual/installation/#mongodb-community-edition-installation-tutorials">
+                                    MongoDB Community Edition{' '}
+                                </StyledParagraphLink>{' '}
+                                to build your own O-FISH instance.
+                            </BodyText>
+
+                            <StyledLink href="https://o-fish.github.io">
+                                Start building now
+                            </StyledLink>
+                        </VesselAppTextContainer>
+
+                        <VesselAppTextContainer>
+                            <ArticleH2>
+                                Contribute code to help protect our oceans
+                            </ArticleH2>
+                            <BodyText>
+                                Submit new features, translations, and styles to
+                                O-FISH. This is an open source application;
+                                anyone can contribute.
+                            </BodyText>
+
+                            <StyledLink href="#">
+                                Contribute your code
+                            </StyledLink>
+                        </VesselAppTextContainer>
+
+                        <VesselAppTextContainer>
```

Comment:
Lets have all of the text containers go full width on mobile, no need to constrain them to the max width at that point